tools/version_utils.py

```diff
@@ -6,9 +6,9 @@
 MAJOR = 1
 MINOR = 9
 MICRO = 0
-ISRELEASED = True
+ISRELEASED = False
```

Comment:
Maybe I don't understand what this is for, should be left to True no?

Reply:
The comment says "final unreleased", so looks like that's on purpose. CI can complete, maybe last checks like sdist permissions etc., before pushing the commit for the release.

Reply:
Yeah, I guess that's what I typically do--switch to `True` when the local steps have "passed" during release process (in this case, probability of finding something new is indeed probably low though).

Reply:
ah ok, I did not understand the docstrings.